test/unit/renderer/viz/expressions/classification/classifier.test.js

```diff
@@ -0,0 +1,265 @@
+import {
+    validateDynamicTypeErrors,
+    validateStaticType,
+    validateStaticTypeErrors,
+    validateCompileTypeError,
+    validateMaxArgumentsError
+} from '../utils';
+
+import { average, standardDeviation } from '../../../../../../src/renderer/viz/expressions/stats';
+import {
+    property,
+    globalQuantiles,
+    globalEqIntervals,
+    globalMeanStandardDev,
+    viewportQuantiles,
+    viewportEqIntervals,
+    viewportMeanStandardDev
+} from '../../../../../../src/renderer/viz/expressions';
+
+import Metadata from '../../../../../../src/renderer/Metadata';
+
+describe('src/renderer/viz/expressions/classifier', () => {
+    describe('error control', () => {
+        describe('global', () => {
+            validateStaticTypeErrors('globalQuantiles', []);
```

Comment:
With one my last PRs this is not going to work anymore, but it's easy to fix, just use the function `validateTypeErrors` after merging with `master`

Reply:
Ok @davidmanzanares  so all `validateStaticTypeErrors` should be replaced with `validateTypeErrors`. 
And what about:
* `validateDynamicTypeErrors`?
* `validateCompileTypeError`?

Reply:
they should too, you can look at https://github.com/CartoDB/carto-vl/pull/917/files